user/tests/integration/communication/startup-slos/p1-and-photon-free-ram-slo.spec.js

```diff
@@ -0,0 +1,12 @@
+// Why this different SLO?
+// The official Device OS calls for 60 kB of free RAM, however, P1 and Photon's have never met that quota and won't
+// so we exclude it here.
+suite('startup free ram SLO for photon and p1');
+platform('photon', 'p1');
+test('slo startup stats', async function () {
+    const target = 45000;
```

Comment:
Let's keep a single spec file for a single suite, otherwise we have code duplication and it's easy to miss things like this when implementing changes later or refactoring. It's possible to get the current DUT from inside the test and use separate limits for specific platforms e.g.:
```js
let target = 60000;
const dut = this.particle.devices[0];
if (dut.platform.name == 'photon' || dut.platform.name == 'p1') {
    target = 45000;
}
```